client/my-sites/migrate/controller.js

```diff
@@ -1,39 +1,18 @@
-import { isEnabled } from '@automattic/calypso-config';
 import { translate } from 'i18n-calypso';
-import page from 'page';
-import { decodeURIComponentIfValid } from 'calypso/lib/url';
-import SectionMigrate from 'calypso/my-sites/migrate/section-migrate';
 import { getSiteId } from 'calypso/state/sites/selectors';
-
-export function ensureFeatureFlag( context, next ) {
-	if ( isEnabled( 'tools/migrate' ) ) {
-		return next();
-	}
-	page.redirect( '/' );
-}
+import SectionMigrate from './section-migrate';
 
 export function migrateSite( context, next ) {
-	if ( isEnabled( 'tools/migrate' ) ) {
-		const sourceSiteId =
-			context.params.sourceSiteId &&
-			getSiteId( context.store.getState(), context.params.sourceSiteId );
-		const fromSite =
-			( context.query &&
-				context.query[ 'from-site' ] &&
-				decodeURIComponentIfValid( context.query[ 'from-site' ] ) ) ||
```

Comment:
Hm, what's with the `decodeURIComponentIfValid` removal?

Reply:
Oh I forgot to mention that in the PR description! The `context.query` items are always already decoded, we do that here through `URLSearchParams`: https://github.com/Automattic/wp-calypso/blob/trunk/client/boot/common.js#L59